sklearn/feature_selection/correlation_threshold.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
+from scipy.sparse import issparse
+
+from ..base import BaseEstimator
+from .base import SelectorMixin
+from ..utils import check_array
+from ..utils.validation import check_is_fitted
+from ..utils.extmath import safe_sparse_dot
+from ..utils.sparsefuncs import min_max_axis, mean_variance_axis
+
+
+class CorrelationThreshold(BaseEstimator, SelectorMixin):
+    """Feature selector that removes features with high correlation.
+
+    This unsupervised feature selection algorithm considers feature pairs with
+    correlation higher than the `threshold` and removes the feature that has
+    the highest mean correlation with the other features. This process is
+    continued till there are no correlations higher than `threshold`.
+
+    Read more in the :ref:`User Guide <correlation_threshold>`.
+
+    Parameters
+    ----------
+    threshold : float, default=0.9
+        Features with a training-set correlation higher than this threshold
+        will be removed.
+
+    Attributes
+    ----------
+    support_mask_ : bool ndarray of shape (n_features, )
```

Comment:
are we doing the space after `,`?

Reply:
We are not. It is one of my habits bad habits.